src/repairnator-pipeline/src/main/java/fr/inria/spirals/repairnator/process/step/repair/AbstractRepairStep.java

```diff
@@ -35,12 +35,16 @@
 public abstract class AbstractRepairStep extends AbstractStep {
 
     public static final String DEFAULT_DIR_PATCHES = "repairnator-patches";
-    public static final String TEXT_PR = "This patch fixes failing Travis build %(travisURL) \n\n" +
+    public static final String DEFAULT_TEXT_PR = "This patch fixes failing Travis build %(travisURL) \n\n" +
```

Comment:
we need different text depending on the kind of repair, so we would add "SONARQUBE_REPAIR_PR_TEXT"

Reply:
It is provided by `GitHub_TEXt_PR` as base. The text part of Sonarqube repair is a variable (it depends on the repair rule ) therefore it can't be constant :) .